src/dataLayer/mongo/user-dataLayer.test.js

```diff
@@ -108,3 +110,40 @@ describe('getUser', () => {
     });
   });
 });
+
+describe('deleteUser', () => {
+  it('should delete an existing user', async done => {
+    const result = await createUser({}, {}, validContext);
+    const { accountLinkId } = result;
+    const response = await deleteUser({}, { accountLinkId }, validContext);
+    expect(response).toBeTruthy();
+    expect(response.accountLinkId).toMatch(accountLinkId);
+    const searchResult = await UserModel.findOne({ accountLinkId });
+    expect(searchResult).toBe(null);
+    done();
+  });
+  it('should return with an error message when a user not in db', async done => {
+    try {
+      const response = await deleteUser(
+        {},
+        { accountLinkId: global.idOfLola },
+        validContextForLola
+      );
+    } catch (err) {
+      expect(err).toMatchSnapshot();
+    }
+    done();
+  });
+  it('should return with an error when deleting the user not logged in', async done => {
```

Comment:
could you please rephrase so it's a little more clear what we are testing? Something like `should return an error message when trying to delete another user` or `should only allow deletion of own account` would do.